### PR TITLE
Multihop: skeleton

### PR DESCRIPTION
**Pull request by gangov**

### CODE REVIEW

**Comment by gangov** [comment]
in multihop::swap() for each existing vector of operations are we calling pair::swap()?

File: contracts/multihop/src/contract.rs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            // ``` fn swap(
+            //         env: Env,
+            //         sender: Address,
+            //         sell_a: bool,
+            //         offer_amount: i128,
+            //         belief_price: Option<i64>,
+            //         max_spread_bps: Option<i64>,
+            //     ) -> Result<(), ContractError>;```
+            // if yes, we'll need more arguments to this function
+        }
```
Comment (ueco-jb) [inline contracts/multihop/src/contract.rs:66]: All unknowns set as None.
Also this PR is supposed to be just a skeleton, just leave it with `unimplemented`.
But add query for admin.

File: contracts/multihop/Makefile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+default: all
+
+all: lint build test
+
+test: build # because of token dependency
```
Comment (ueco-jb) [inline contracts/multihop/Makefile:5]:
```suggestion
test:
```

File: contracts/multihop/src/contract.rs (hunk 1 above)
Comment (gangov) [reply contracts/multihop/src/contract.rs:66]: the get_admin_query is in storage.rs, appended _ so the compiler doesn't complain

**Review by ueco-jb — APPROVED** [review]
LGTM

**Pull request merged**